test/test.js

```diff
@@ -1497,6 +1497,18 @@ describe('Page', function() {
         [200, 300]
       ]);
     }));
+    it('should tap the button', SX(async function() {
+      await page.goto(PREFIX + '/input/button.html');
```

Comment:
is is actually ok to send touches to the non-touch viewport?

Reply:
It seems fine.